### DISCUSSION TITLE
xterm monorepo

### DISCUSSION THREAD

**Discussion by UziTech** [renovatebot/renovate#12563] [Request Help] [answered]
How can we add xterm monorepo so xterm and addons get updated at once?

https://github.com/xtermjs/xterm.js/

**Comment by viceice** [comment, marked as answer]
https://github.com/renovatebot/renovate/blob/ab018191fac6961b5f9d0a73261ac8fba0b4566f/lib/config/presets/internal/monorepo.ts#L4

**Reply by HonkingGoose** [reply]
Resolved by PR: #12580
